### PR TITLE
Token Manager

### PR DESCRIPTION
**Pull request by bpierre**
Normal Mode:

image

Group Mode:

image

### CODE REVIEW

**Comment by izqui** [comment]
Awesome work @bpierre! We can remove everything regarding 'wrapped tokens' as that concept no longer exists :)

**Comment by onbjerg** [comment]
Looks amazing! Where does the token names come from?

**Comment by izqui** [comment]
`Foo` should come from the token symbol which should be provided as app context.

**Comment by sohkai** [comment]
Oh snap, I thought the distribution graph wouldn't make it for this release 🎉!

File: apps/token-manager/app/public/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="theme-color" content="#000000">
+    <title>Aragon Voting</title>
```
Comment (sohkai) [inline apps/token-manager/app/public/index.html]: Aragon Token Manager

File: apps/token-manager/app/src/components/HolderRow.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+class HolderRow extends React.Component {
+  static defaultProps = {
+    name: '',
+    balance: 0,
+    groupMode: false,
+  }
+  static ActionLabel = styled.span`
+    margin-left: 15px;
+  `
```
Comment (sohkai) [inline apps/token-manager/app/src/components/HolderRow.js]: Do you prefer this to having an outside component (or was there another advantage to adding it inside the component)?

This seems more complicated to me, especially with the `Self` reference.

File: apps/token-manager/app/src/components/SideBar.js [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+  }
+}
+
+const Main = styled.aside`
+  flex-shrink: 0;
+  flex-grow: 0;
+  width: 260px;
+  margin-left: 30px;
+  min-height: 100%;
+  li {
```
Comment (sohkai) [inline apps/token-manager/app/src/components/SideBar.js]: Maybe move this down into `InfoRow` and `StakesListItem`?

File: apps/token-manager/app/src/components/HolderRow.js (hunk 2 above)
Comment (bpierre) [reply apps/token-manager/app/src/components/HolderRow.js]: Yes I was trying this, I was waiting for your feedback ha ha 😆

I’ll go back to the previous style.

The idea was to move everything inside the class, to make it clearer when we have multiple components in a single file (`HolderRow` was in `screens/Holders.js`, then I moved it).

But I agree it’s not as readable, and it’s probably a good thing to make multiple component files look bad, so we are pushed to use a single component per file.

Comment (sohkai) [reply apps/token-manager/app/src/components/HolderRow.js]: > Make it clearer when we have multiple components in a single file

I like solving this by just moving out those multiple components into other files :).

**Pull request merged**